### PR TITLE
updated with Defender status

### PR DESCRIPTION
**Pull request by DevOpsStyle**
Hello All,

actually in the documentation was reported the method to disable the Defender with the Registry Key. I aso addedd the procedure for disable the Defender with a GPO.

Thank you all for the support

### CODE REVIEW

**Review by DevOpsStyle — COMMENTED** [review]
Added information about GPO procedure related Defender

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md [hunk 1]
```diff
@@ -98,12 +98,91 @@ The following table summarizes the state of Microsoft Defender Antivirus in seve
...
 - Path: `HKLM\SOFTWARE\Policies\Microsoft\Windows Advanced Threat Protection`
 - Name: `ForceDefenderPassiveMode`
 - Type: `REG_DWORD`
 - Value: `1`
 
+**GPO Method**
+
+- Open Group Policy Management Editor -> Computer Configuration -> Administrative Templates -> Windows Components -> Microsoft Defender Antivirus
+- Select "Turn Off Microsoft Defender Antivirus"
+- Set the GPO to "Enabled"
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md]:
```suggestion
- Open Group Policy Management Editor -> **Computer Configuration** -> **Administrative Templates** -> **Windows Components** -> **Microsoft Defender Antivirus**.
- Select **Turn Off Microsoft Defender Antivirus**.
- Set the GPO to **Enabled**.
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md [hunk 2]
```diff
@@ -98,12 +98,91 @@ The following table summarizes the state of Microsoft Defender Antivirus in seve
...
+
+We can view the Protection status in Powershell with the command "Get-MpComputerStatus" and the key "AMRunningMode"
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md]:
```suggestion
You can view the Protection status in PowerShell with the command "Get-MpComputerStatus" and the key "AMRunningMode".
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md [hunk 3]
```diff
@@ -98,12 +98,91 @@ The following table summarizes the state of Microsoft Defender Antivirus in seve
...
+TroubleShootingEndTime           :
+TroubleShootingExpirationLeft    :
+TroubleShootingMode              :
+TroubleShootingModeSource        :
+TroubleShootingQuotaResetTime    :
+TroubleShootingStartTime         :
+PSComputerName                   :
+```
+
+In the example below the Defender status is **Not Running**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md]:
```suggestion
In the following example, the Defender status is **Not Running**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DevOpsStyle, kindly commit the suggested changes. Many thanks.

**Comment by DevOpsStyle** [comment]
> Copy review - @DevOpsStyle, kindly commit the suggested changes. Many thanks.

Thanks for your support.

Your suggestions are commited

File: microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md [hunk 4]
```diff
@@ -98,12 +98,91 @@ The following table summarizes the state of Microsoft Defender Antivirus in seve
...
+
   You can set Microsoft Defender Antivirus to passive mode by setting the following registry key:
 - Path: `HKLM\SOFTWARE\Policies\Microsoft\Windows Advanced Threat Protection`
 - Name: `ForceDefenderPassiveMode`
 - Type: `REG_DWORD`
 - Value: `1`
 
+**GPO Method**
+
+- Open Group Policy Management Editor -> **Computer Configuration** -> **Administrative Templates** -> **Windows Components** -> **Microsoft Defender Antivirus**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-compatibility.md]:
```suggestion
- Open Group Policy Management Editor > **Computer Configuration** > **Administrative Templates** > **Windows Components** > **Microsoft Defender Antivirus**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
My apologies @DevOpsStyle, just minor style changes that I missed the first time around. Thank you so much.

**Comment by DevOpsStyle** [comment]
> My apologies @DevOpsStyle, just minor style changes that I missed the first time around. Thank you so much.

Please don't say sorry :) 
Thanks again for your help, commit pushed 👍

**Review by JohanFreelancer9 — APPROVED** [review]
Thanks, @DevOpsStyle.

**Comment by DevOpsStyle** [comment]
> Thanks, @DevOpsStyle.

Thank you too :) 
There is nothing else i need to do previously to the merge?

**Review by denisebmsft — APPROVED** [review]
Thank you @DevOpsStyle

**Pull request merged**